### DISCUSSION TITLE
relative_path option includes all folders using full path

### DISCUSSION THREAD

**Discussion by videositefree** [spatie/laravel-backup#1413] [Q&A]
Hi,
I`m try change path in created zip folder in the backup.php config:

i`m try:
'relative_path' => base_path()
OR
'relative_path' => storage_path('app/backup-temp/temp')
OR
'relative_path' => base_path('app/backup-temp/temp')

use 
\Artisan::call('backup:run',['--only-db'=>true]);

create zip folder and have absolute path
c:\and full path where is site. aboute 11 folder

how change for relative path?

**Comment by erikn69** [comment]
**`c:\`**?? that means windows

this package is developed for linux environments, in windows it may fail or not all the features are available, but the basic works

**Reply by videositefree** [reply]
thanks for answer, 
earlier version don`t have this problem in windows and xampp. :)

**Reply by erikn69** [reply]
What version do you have?, and wich version works?? Maybe i can fix it
but as I said, it is not focused on windows

**Reply by erikn69** [reply]
I run test on windows, `relative_path` works as expected
https://github.com/spatie/laravel-backup/blob/8a14d26cc10d0372e9206594c26cc56ff15997ac/tests/Commands/BackupCommandTest.php#L94-L105
![image](https://user-images.githubusercontent.com/4933954/135870537-ce8257a3-f743-416f-9dd6-ef1ddb2484bd.png)

**Comment by videositefree** [comment]
If use comand 
\Artisan::call('backup:run',['--only-db'=>true]);

![1](https://user-images.githubusercontent.com/62680572/135878693-b1e8ea5b-2635-471d-b550-581138389850.PNG)

and last folder in global path
![2](https://user-images.githubusercontent.com/62680572/135878939-cd7acf73-fd49-4b10-851d-ecb6aac9b385.PNG)

**Reply by erikn69** [reply]
>What version do you have?, and wich version works??

Tested `--only-db` with V6 and V7 current versions on windows
![image](https://user-images.githubusercontent.com/4933954/135886501-0313db9c-ee1f-431e-aac5-fc2131673953.png)
